lib/queue.js

```diff
@@ -276,6 +276,37 @@ class Queue extends Emitter {
     });
   }
 
+  _getJobsByIds(ids) {
+    // If ids is a Set (i.e. returned from Queue._scanForJobs), convert it to an
+    // Array.
+    const idsArray = Array.from(ids);
+    if (!idsArray.length) {
+      return Promise.resolve([]);
+    }
+
+    // We need to re-ensure the queue is commandable, as we might be shutting
+    // down during this operation.
+    return this._commandable().then((client) => {
+      const got = helpers.deferred();
+      const commandArgs = [this.toKey('jobs')].concat(idsArray, got.defer());
```

Comment:
My specific thinking here is that this could have some benefit in the use case of "arena is sitting here calling getJobs() every 10 seconds to keep an up-to-date dashboard display of jobs that have failed". I have no idea if arena in particular actually does something like that or if the intent is to use it like that, but it seems like a reasonable monitoring possibility.